handbook: Remove PIPs (#4309)
FlowFuse/website: src/handbook/peopleops/coaching-plans.md

# Coaching Plans

> A Coaching Plan is a tool for managers to aid in development of their team

Reviewer: ```suggestion
A Coaching Plan is a tool managers use to support a team member’s development.
It helps them grow in their role through a structured plan, built and completed together with their manager’s mentorship.
```

# Coaching Plans

A Coaching Plan is a tool for managers to aid in development of their team
members, helping them grow in their role and improve their performance through a
structured plan to be completed with the manager's mentorship. Coaching plans can
be used both proactively for career development and when addressing performance
concerns, always with a focus on growth, support, and clear expectations. 

## Purpose and Philosophy

Coaching Plans are built on the belief that all team members have potential for growth — in their job skills, alignment with company values, or execution of their role.

Coaching Plans are a positive, intentional investment. They are used to help team members prepare for career advancement, role changes, and evolving expectations as the company grows.

## When to Use a Coaching Plan

> A coaching plan should be initiated by a team member's manager when:

Reviewer: ```suggestion
A Coaching Plan may be initiated by a team member’s manager when:
```

- A team member shows potential for growth
- The growth area is specific and coachable

> - Performance issues need to be addressed with clear expectations and support

Reviewer: ```suggestion
- Expectations need to be clarified and supported through structured development
```

- Minor performance areas need attention and structured improvement
- A team member expresses interest in expanding their responsibilities or transitioning to a new role with additional responsibilities

> - The team member is to be prepared for future role progression

Reviewer: ```suggestion
- A team member is preparing for future role progression
```

> - You observe behaviors or skills that could benefit from focused development

Reviewer: ```suggestion
- A manager observes behaviors or skills that could benefit from intentional development
```

  - Coaching plans are only shared with: the team member, manager, the managers'

>     manager, and HR

Reviewer: ```suggestion
  - Coaching Plans are only shared with the team member, their manager, the manager’s manager, and HR
```
